sample_go_module/README.md

```diff
@@ -32,90 +32,65 @@ To setup your own project to build modules for the game server you can follow th
 
 1. Download and install the Go toolchain. It's recommended you follow the [official instructions](https://golang.org/doc/install).
 
-2. Setup your GOPATH environment variable. Most use `$HOME/go` as the `$GOPATH`.
-
-   You can temporarily setup the environment variable with `export` but for it to persist you should add it to your shell environment.
-
-   ```
-   export GOPATH=$HOME/go
-   ```
-
-3. Use "go get" to download the server locally.
-
-   ```
-   go get -d github.com/heroiclabs/nakama
-   ```
-
-4. Build the game server from source if you want.
-
-   ```
-   cd $GOPATH/src/github.com/heroiclabs/nakama
-   env CGO_ENABLED=1 go build -trimpath
-   ```
+2. Setup a folder for your own plugin code.
 
-5. Setup a folder for your own server code.
+    ```bash
+    mkdir -p $HOME/my-plugin-project
+    cd $HOME/my-plugin-project
+    ```
 
-   ```
-   mkdir -p $GOPATH/src/some_project
-   cd $GOPATH/src/some_project
-   go get -u "github.com/heroiclabs/nakama-common"
-   ```
+3. Init the go module for your plugin and add the nakama-common dependency.
 
-6. You'll need to copy the main server dependencies into your project.
+    ```bash
+    go mod init my-plugin-project
+    go get -u github.com/heroiclabs/nakama-common@v1.0.0
```

Comment:
```
go get -u "github.com/heroiclabs/nakama-common@v1.0.0"
```

Probably best to quote the go get arguments for semver in case of exotic shells.